common/lib/api-client/models.test.js

```diff
@@ -25,17 +25,20 @@ const testCases = {
       httpMock: 'get',
       mockPath: '/1',
       args: '1',
+      statusCode: 200,
     },
     {
       method: 'findAll',
       httpMock: 'get',
+      statusCode: 200,
     },
   ],
   person: [
     {
       method: 'create',
       httpMock: 'post',
       args: {},
+      statusCode: 200,
```

Comment:
Now we have this support we should actually look to use the correct status code for created resources. POSTs should return a `201` status.

Reply:
Good point